src/Paysera/WalletApi/Entity/Client.php

```diff
@@ -265,4 +270,23 @@ public function setServiceAgreementId($serviceAgreementId)
 
         return $this;
     }
+
+    /**
+     * @return Paysera_WalletApi_Entity_ClientWalletPermissions[]
+     */
+    public function getWalletsPermissions()
+    {
+        return $this->walletsPermissions;
+    }
+
+    /**
+     * @param Paysera_WalletApi_Entity_ClientWalletPermissions[] $walletsPermissions
+     * @return $this
+     */
+    public function setWalletsPermissions($walletsPermissions)
```

Comment:
if there is a variable with set permissions, then we should also have the option to add by one entry.

Another question what does it mean walletsPermissions? it's a bit unclear what it sets.

It looked like Client Permissions to the specific wallet, so maybe could be like: ClientPermissionsToWallets, ClientPermissionsByWallets... what do you think?